scripts/popup/controller/TestStep.controller.js

```diff
@@ -1013,8 +1040,9 @@ sap.ui.define([
         _updatePreview: function () {
             var oItem = this._oModel.getProperty("/element");
             this._suspendBindings();
-            this._adjustBeforeSaving(oItem).then(function (oElementFinal) {
-                this._getFoundElements().then(function (aElements) {
+            return this._adjustBeforeSaving(oItem).then(function (oElementFinal) {
+                //can not understand why we need this, here.
```

Comment:
🐛 Unfortunately, you have introduced a regression by commenting out this code part: The bindings are suspended above but are not resumed anymore now, such that the on-the-fly execution of asserts and the checking of actions do not work anymore. See the exemplary screenshot below.

Please execute `this._resumeBindings();` below again to fix this regression.

--------------

### Example
While testing <https://ui5.sap.com/#/api/sap.m>, I selected a `DemokitTreeItem` control on the left and did not specify "a unique combination" in the test recorder then. After configuring an assert, the assert's result (it is obviously failing) is not shown immediately via the UI (star rating is missing plus "Found items (325)" is not colored red).

![annotation-assert](https://user-images.githubusercontent.com/58171110/79853824-d779c100-83c8-11ea-859d-e859a7933934.png)